api/v1alpha1/osbuildconfig_types.go

```diff
@@ -53,7 +53,7 @@ type ParameterValue struct {
 	Value string `json:"value"`
 }
 
-// BuildDetails includes all the information needed to build the image
+// BuildDetails includes all the information needed to build the edge-container image
```

Comment:
As far as the user who creates the OSBuildConfig CR is concerned BuildDetails is the same for all image types. It's not specific to edge-container. Please revert